lib/stat_tracker.rb

```diff
@@ -33,10 +31,12 @@ def self.teams_reader(csv_data)
     
     def self.game_teams_reader(csv_data)
         seasons = Hash.new(0)
+        count = 1 
         CSV.foreach(csv_data, headers: true, header_converters: :symbol) do |row|
-            seasons[row[:game_id].to_i] = Season.new(row)
+            seasons[count] = Season.new(row)
```

Comment:
Great way to bypass this bug and access all rows even if they share a game_id.

Reply:
I second this. It found a work around to the data coming to us.